frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx

```diff
@@ -47,12 +47,20 @@ const EXAMPLES = {
     },
 }
 
+
 const TagExample = ({ datasetQuery, setQuery }) =>
     <div>
         <h5>Example:</h5>
         <p>
             <Code>{datasetQuery.native.query}</Code>
-            { setQuery && <div className="Button Button--small" onClick={() => setQuery(datasetQuery, true)}>Try it</div> }
+            { setQuery && (
```

Comment:
Looks like you removed the actual call to setQuery, which is definitely not what we want?